javascripts/discourse/api-initializers/table-editor.js

```diff
@@ -35,12 +39,26 @@ export default apiInitializer("0.11.1", (api) => {
 
     return ajax(`/posts/${this.id}`, { type: "GET" })
       .then((post) => {
-        showModal("insert-table-modal", {
-          model: post,
-        }).setProperties({
-          tableHtml: tempTable,
-          tableId,
-        });
+        let markdownItURL = Session.currentProp("markdownItURL");
+
+        if (markdownItURL) {
+          loadScript(markdownItURL).then(() => {
+            const prettyText = new PrettyText(buildOptions({ siteSettings }));
+            const tokens = prettyText.opts.engine.parse(post.raw, {
+              references: {},
+            });
+            const allTables = tokenRange(tokens, "table_open", "table_close");
+            const tableTokens = allTables[tableId];
+
+            showModal("insert-table-modal", {
+              model: post,
+            }).setProperties({
+              tableHtml: tempTable,
```

Comment:
This is no longer needed, right?

Reply:
I am still using it here:
```
  get isEditingTable() {
    if (this.args.tableHtml) {
      return true;
    }

    return false;
  }
```

as I was having some issues when using `this.args.tableTokens` instead, but I'll be trying to get rid of it.